README.md

```diff
@@ -30,6 +30,9 @@ This app is still in the early stages of development and only some functionality
 ---
 
 # Local Development
+<<<<<<< HEAD
```

Comment:
This was a bad merge conflict resolution. Have you ever had to resolve a merge conflict before?
```suggestion
```

Reply:
Yes. I had to resolve a merge conflict before. I will make the suggested changes.